### DISCUSSION TITLE
Welcome to flatcc Discussions!

### DISCUSSION THREAD

**Discussion by mikkelfj** [dvidelabs/flatcc#167] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JohanNicander** [comment]
Hello!

Is there a new release planned for flatcc any time soon?

**Reply by mikkelfj** [reply]
Well, a release is long over due, I just have a lot on my plate. Also, I'd like to a have quiet period after changes that might introduce regressions. But I'd say a release should come out in the next few months.
